NuGetPackageVerifier.json

```diff
@@ -0,0 +1,134 @@
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Microsoft.Framework.NotNullAttribute.Sources.dll": "BUG: This file shouldn't be here",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\Microsoft.CodeAnalysis.CSharp.dll": "Need to follow up, not from ADX team",
+                    "bin\\Microsoft.CodeAnalysis.dll": "Need to follow up, not from ADX team",
+                    "bin\\Microsoft.Framework.HashCodeCombiner.Sources.dll": "BUG: This file shouldn't be here",
+                    "bin\\mscorlib.dll": "Need to follow up, not from ADX team",
+                    "bin\\System.Security.SecureString.dll": "Need to follow up, not from ADX team"
+                }
+            },
+            "dnx-coreclr-win-x86": {
+                "VERSION_INFORMATIONALVERSION": {
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Newtonsoft.Json.dll": "3rd party DLLs don't need this"
+                },
+                "SERVICING_ATTRIBUTE": {
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Microsoft.Framework.NotNullAttribute.Sources.dll": "BUG: This file shouldn't be here",
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Microsoft.Framework.NotNullAttribute.Sources.dll": "BUG: This file shouldn't be here",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\Microsoft.CodeAnalysis.CSharp.dll": "Need to follow up, not from ADX team",
+                    "bin\\Microsoft.CodeAnalysis.dll": "Need to follow up, not from ADX team",
+                    "bin\\Microsoft.Framework.HashCodeCombiner.Sources.dll": "BUG: This file shouldn't be here",
+                    "bin\\mscorlib.dll": "Need to follow up, not from ADX team",
+                    "bin\\System.Security.SecureString.dll": "Need to follow up, not from ADX team"
+                }
+            },
+            "dnx-mono": {
+                "VERSION_INFORMATIONALVERSION": {
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\lib\\Microsoft.Dnx.Tooling\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
+                    "bin\\Microsoft.Dnx.Host.Mono.dll": "Need to follow up, not from ADX team"
+                },
+                "VERSION_FILEVERSION": {
+                    "bin\\Microsoft.Dnx.Host.Mono.dll": "Need to follow up, not from ADX team"
```

Comment:
:+1: There is and issue for this https://github.com/aspnet/dnx/issues/2045. Will update the  justification.